app/js/script.js

```diff
@@ -12,3 +12,36 @@ icons.forEach(icon => {
     bodyContainer.classList.toggle("c-main--shift-left");
   });
 });
+
+document
```

Comment:
I think it'd be useful to add a comment above each block to just describe what it's doing, i.e. 

`// Handle open/close of burger menu`

or similar